networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -13,6 +13,11 @@ These release notes track the development of the AWS Load Balancer Operator in {
 
 For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
 
+[id="aws-load-balancer-operator-release-notes-1.1.1"]
+== AWS Load Balancer Operator 1.1.1
+
+The following advisory is available for the AWS Load Balancer Operator version 1.1.1: link:https://access.redhat.com/errata/RHEA-2024:0555[RHEA-2024:0555 Release of AWS Load Balancer Operator 1.1.z on OperatorHub]
```

Comment:
I just noticed that the link to the advisory should be a bullet point to be consistent with the previous release notes.